vk_ndk_wrapper_include/kompute_vk_ndk_wrapper.cpp

```diff
@@ -15,782 +15,1111 @@
  */
 // This file is generated.
 #ifdef __cplusplus
-extern "C" {
```

Comment:
This one as well (the vk_ndk_wrapper include)

Reply:
Done.